openapi.yaml

```
@@ -10522,6 +10522,64 @@ paths:
       operationId: cleanFailedJob
       x-stoplight:
         id: eu77thczavme3
+  '/environment/{environmentId}/lifecycleTemplate':
+    parameters:
+      - schema:
+          type: string
+        name: environmentId
+        in: path
+        required: true
+    get:
+      summary: Your GET endpoint
+      tags: []
+      operationId: get-environment-environmentId-lifecycleTemplate
```

To be coherent with existing ones in camelcase + better name:
```suggestion
      operationId: getEnvironmentLifecycleTemplates
```